src/Microsoft.Extensions.Options/IOptionsConfiguration.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Options
+{
+    public interface IOptionsConfiguration<in TOptions> where TOptions : class
+    {
+        void Configure(TOptions options);
```

Comment:
We should consider revisiting the name of this method if we agree that we don't want to overload "configure" and "configuration".